bedrock/firefox/views.py

```diff
@@ -613,14 +612,7 @@ def ios_testflight(request):
 
 
 def features_landing(request):
```

Comment:
Does this URL need to be a view still, could it just be a regular page defined in the URLs file?